RetailCoder.VBE/UI/Command/AddTestMethodExpectedErrorCommand.cs

```diff
@@ -78,29 +80,46 @@ protected override bool EvaluateCanExecute(object parameter)
             }
         }
 
-        protected override void OnExecute(object parameter)
+        private bool HasEqualCodeModule(QualifiedModuleName module, ICodeModule otherCodeModule)
```

Comment:
this is the second time I see this helper method defined... Should we get rid of that duplication, and if so, how?